script/results.html

```diff
@@ -3,36 +3,94 @@
+        <button class="nav-link" id="metadata-tab" data-bs-toggle="tab" data-bs-target="#metadata" type="button" role="tab" aria-controls="metadata" aria-selected="false" onclick="disableButton2()">Metadata</button>
       </li>
       <li class="nav-item">
-        <button class="nav-link" id="nodes-tab" data-bs-toggle="tab" data-bs-target="#nodes" type="button" role="tab" aria-controls="nodes" aria-selected="false">Nodes</button>
+        <button class="nav-link" id="nodes-tab" data-bs-toggle="tab" data-bs-target="#nodes" type="button" role="tab" aria-controls="nodes" aria-selected="false" onclick="disableButton2()" >Nodes</button>
       </li>
       <li class="nav-item">
-        <button class="nav-link" id="results-tab" data-bs-toggle="tab" data-bs-target="#results" type="button" role="tab" aria-controls="results" aria-selected="false">Results</button>
+        <button class="nav-link" id="result-tab" data-bs-toggle="tab" data-bs-target="#results" type="button" role="tab" aria-controls="nodes" aria-selected="false" onclick="enableoptionbutton()">Results</button>
       </li>
       <li class="nav-item">
-        <button class="nav-link" id="versions-tab" data-bs-toggle="tab" data-bs-target="#versions" type="button" role="tab" aria-controls="versions" aria-selected="false">Versions</button>
+        <button class="nav-link" id="versions-tab" data-bs-toggle="tab" data-bs-target="#versions" type="button" role="tab" aria-controls="versions" aria-selected="false" onclick="disableButton2()" >Versions</button>
       </li>
-    </ul>
+      <li class="nav-item">
+        <button class="nav-link" id="feedback-tab" data-bs-toggle="tab" data-bs-target="#feedback" type="button" role="tab" aria-controls="nodes" aria-selected="false" onclick="enableoptionbuttonFeedback()" >Feedback</button>
+      </li>
+
+
+        </ul>
+        <ul class="nav nav-tabs">
+          <li class="nav-item">
+            <label  id="myCheck1" for="optional-tab" hidden="hidden" >Show Optional Results for that category</label> 
+            <input type="checkbox"  id="optional-tab"  data-bs-target="#results" data-bs-toggle="tab" role="tab"    onclick="myFunction()" hidden="hidden" >  
+            <label  id="myCheck2" for="optional-tab-non-telco" hidden="hidden" >Show Optional Results for that category</label> 
+            <input type="checkbox"  id="optional-tab-non-telco"  data-bs-target="#results" data-bs-toggle="tab" role="tab"    onclick="myFunction2()" hidden="hidden" >   
+            <label  id="myCheck3" for="optional-tab-telco" hidden="hidden" >Show Optional Results for that category</label> 
+            <input type="checkbox"  id="optional-tab-telco"   data-bs-target="#results" data-bs-toggle="tab" role="tab"  onclick="myFunction3()" hidden="hidden" >   
+              <label  id="myCheck1-feedback" for="optional-tab-feedback" hidden="hidden" >Show Optional feedback for that category</label> 
```

Comment:
Stay consistent in wording "Show Optional feedback for that category" should be  "Show Optional feedback for that scenario"